topi/include/topi/nn.h

```diff
@@ -83,6 +84,38 @@ inline tvm::Tensor leaky_relu(const tvm::Tensor& t,
     tag);
 }
 
+/*!
+ * \brief Creates an operation that performs a parametric rectified linear unit
+ *
+ * \param x The input data tensor
+ * \param slope The channel-wise slope tensor
+ * \param layout The layout of input data, NCHW or NHWC
+ * \param name The name of the operation
+ * \param tag The tag to mark the operation
+ *
+ * \return A Tensor whose op member is the relu operation
+ */
+template <typename T>
+inline tvm::Tensor prelu(const tvm::Tensor &x, const tvm::Tensor &slope,
+                         const std::string &layout = "NCHW",
```

Comment:
since we are only on channel, change layout argument to be channel_axis argument(which indicates the channel axis)

Reply:
so it works for all layouts